dasharo-security/secure-boot.robot

```diff
@@ -233,6 +234,57 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Check automatic certificate provisioning
+    [Documentation]    This test verifies that the automatic certificate
+    ...    provisioning will install custom keys which will make Ubuntu
+    ...    unbootable.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO009.001 not supported
+    Power On
+    # Erase Secure Boot Keys
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    Enable Secure Boot    ${sb_menu}
+    Save Changes
+    Reenter Menu
+    ${sb_menu}=    Get Secure Boot Menu Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    # Reboot to Automatic provisioning
+    Boot Dasharo Tools Suite    USB    True
+    Boot System Or From Connected Disk    ubuntu
+
+    Read From Terminal Until    Press any key to continue...
+
+SBO009.002 Check automatic certificate provisioning KEK certificate
+    [Documentation]    This test verifies that the automatic certificate
+    ...    provisioning installs the expected KEK certificate.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO009.002 not supported
+    Power On
+    Boot Dasharo Tools Suite    USB
+    Enter Shell In DTS
+    # Get the KEK certificates
```

Comment:
@PLangowski it would be better to have keyword that do such comparison.

Reply:
@TomaszAIR The keyword `Compare Mokutil And OpenSSL Outputs` already exists and is being used in this test.

Reply:
@DaniilKl maybe it could also get KEK.crt, output from mokutil --kek and use openssl so we would not need to run it in this test steps? One keyword would be much more readable

Reply:
Amended here 17566474991b6d643d97065e34d0c49e96d1fa9b and here c2117381563a4894b78a6d5e72217ae4600e0800.